packages/api/src/graphql/subscription.ts

```diff
@@ -50,7 +50,7 @@ export const GraphQLSubscription = new GraphQLObjectType<Subscription, Context>(
     createdAt: {type: GraphQLNonNull(GraphQLDateTime)},
     modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
     user: {
-      type: GraphQLNonNull(GraphQLUser),
```

Comment:
Why is here not Null constraint removed. Doesn't a subscription always belong to a user?

Reply:
Once an user gets deleted, the subscription is null. otherwise the app fails. I opened a ticket, so that no user gets hardly deleted from the database but are gonna be flagged.